components/ui/button/utils/helpers.ts

```diff
@@ -32,7 +32,7 @@ export function buttonSizes(size?: ButtonSizes) {
       return "px-6 py-3 text-base";
     }
     default: {
-      return "px-4 py-2 text-sm";
+      return "px-4 py-2 text-base sm:text-sm";
```

Comment:
This shouldn't be changed as this will cause the button to have different height for the button.

![image](https://user-images.githubusercontent.com/5663877/127213228-df195d81-a40f-4f64-a606-b9cbd13e4295.png)

Reply:
Reverted